### ISSUE TITLE
More extensive feedback feature

### ISSUE THREAD

**Issue by PhictionalOne** [infomark-org/infomark-ui#8]
The single line of Feedback is in my opinion not enough or better not the best way to present feedback. As with a long extensive comment the readability suffers a lot.

I'd personally prefer the option to upload a modified version of the received handin with comments. But this might not be feasable.

An other option I'd propose is the possibility to blow up the current Feedback system by binding the oneline feedback comment to a subtask of the assignment. And enabling multiple feedbacks on one assignment.

For example:
```
[ a) ][ Type Feedback here...               ]
[ +  ]
------------------o----------------- 6 Points
[                  Grade                    ]
```

Maybe a Dropdown menu to choose the subtask.
```
| general Feedback |
>        a)        <
|        b)        |
|        ...       |
```

**Comment by PatWie** [comment]
You can use Markdown for feedback

**Comment by kryonix** [comment]
Fixed #12.
